src/awkward/operations/ak_broadcast_arrays.py

```diff
@@ -126,7 +126,7 @@ def broadcast_arrays(
             output = []
             for inner in outer:
                 output.append(x + inner)
-            yield output
+            yield arrayput
```

Comment:
Did you find-and-replace `out` → `array` to get "`arrayput`"?

Reply:
Ha, yes, but not in the same pass. This one snuck in despite a thorough check (not thorough enough ...)